### DISCUSSION TITLE
Disable random plugin installs

### DISCUSSION THREAD

**Discussion by Android789515** [LunarVim/LunarVim#4470] [Q&A] [answered]
I just opened up a tsx file and LunarVim started installing tailwindcss, I don't want that on my system.  How do I turn this off?

![2024-01-16-22:05:11-screenshot](https://github.com/LunarVim/LunarVim/assets/90219092/2d89ebfa-65b8-474a-8461-faac6c5780f2)

**Comment by juicy-g** [comment, marked as answer]
`tsserver` and `tailwindcss` are LSP servers and are the default servers for `tsx` files rather than just some random plugins.  

If you really want to disable them you can put this line in your config:
```lua
lvim.lsp.automatic_servers_installation = false
```
If you just want to just disable `tailwindcss` then you can do:
```
vim.list_extend(lvim.lsp.automatic_configuration.skipped_servers, { "tailwindcss" })
```
Any changes to `lvim.lsp.automatic_configuration.skipped_servers` must be followed by `:LvimCacheReset` to take effect.

I suggest you read this page of the docs for more information: [https://www.lunarvim.org/docs/configuration/language-features/language-servers](https://www.lunarvim.org/docs/configuration/language-features/language-servers).
